.github/workflows/release-operator.yml

```diff
@@ -303,6 +323,47 @@ jobs:
           push: true
           tags: "${{ env.OPERATOR_IMAGE_REPOSITORY }}:latest-${{ steps.setup-build-step.outputs.platform_os }}-${{ steps.setup-build-step.outputs.platform_arch }},${{ env.OPERATOR_IMAGE_REPOSITORY }}:${{ env.OPERATOR_VERSION }}-${{ steps.setup-build-step.outputs.platform_os }}-${{ steps.setup-build-step.outputs.platform_arch }}"
 
+      - name: Cosign login and initialize
+        run: |
+          echo "${{ secrets.REGISTRY_PASSWORD }}" | cosign login --username ${{ secrets.REGISTRY_USERNAME }} --password-stdin ${OPERATOR_IMAGE_REGISTRY%%/*}
+          cosign initialize
```

Comment:
Is initialize required if we are using the public good instance of sigstore?

Reply:
done